users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -918,13 +1136,11 @@ def py():
         columns_start=["train_name", "wave_norm", "specaug", "lr", "batch_size"],
         columns_end=["epoch", "recog_name", "lm", "optlm", "lm_scale", "prior_scale"],
     )
-    report = Report.merge_reports(
-        [
-            report_base,
-            report_mel,
-            report_scf,
-        ]
-    )
+    report = Report.merge_reports([
+        report_base,
+        report_mel,
+        report_scf,
+    ])
```

Comment:
This looks like reverting the black changes. Please apply black to the file and check for these unnecessary changes.